File: fix_colors.js
const fs = require('fs');

const filePath = 'c:\\Users\\Cristiano D. Borges\\Downloads\\Plataforma Growth Summit 2026\\app\\src\\pages\\public\\GrowthExperienceTriunfo.tsx';

// Ler o arquivo
const content = fs.readFileSync(filePath, 'utf8');

// Contar ocorrências antes
const countBefore = (content.match(/brand-yellow/g) || []).length;
console.log(`Encontradas ${countBefore} ocorrências de 'brand-yellow'`);

// Substituir
const newContent = content.replace(/brand-yellow/g, 'brand-orange-coral');

// Contar ocorrências depois
const countAfter = (newContent.match(/brand-yellow/g) || []).length;
console.log(`Restam ${countAfter} ocorrências de 'brand-yellow'`);

// Salvar
fs.writeFileSync(filePath, newContent, 'utf8');

console.log(`✅ Substituição concluída! ${countBefore} ocorrências foram substituídas.`);

File: fix_colors.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""Script para substituir brand-yellow por brand-orange-coral"""

import os

file_path = r"c:\Users\Cristiano D. Borges\Downloads\Plataforma Growth Summit 2026\app\src\pages\public\GrowthExperienceTriunfo.tsx"

# Ler o arquivo
with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()

# Contar ocorrências antes
count_before = content.count('brand-yellow')
print(f"Encontradas {count_before} ocorrências de 'brand-yellow'")

# Substituir
content = content.replace('brand-yellow', 'brand-orange-coral')

# Contar ocorrências depois
count_after = content.count('brand-yellow')
print(f"Restam {count_after} ocorrências de 'brand-yellow'")

# Salvar
with open(file_path, 'w', encoding='utf-8') as f:
    f.write(content)

print(f"✅ Substituição concluída! {count_before} ocorrências foram substituídas.")
